test/integration/service.spec.ts

```diff
@@ -68,6 +71,118 @@ test('when sending a request message', args => {
   })
 })
 
+test(`when sending a request message for a method that is not ${METHOD_DCL_PERSONAL_SIGN}`, args => {
+  let mainAccount: ethers.HDNodeWallet
+  let ephemeralAccount: ethers.HDNodeWallet
+  let expiration: Date
+  let ephemeralMessage: string
+  let signature: string
+  let authChain: AuthChain
+
+  beforeEach(async () => {
+    await connectClients(args)
+
+    mainAccount = ethers.Wallet.createRandom()
+    ephemeralAccount = ethers.Wallet.createRandom()
+    expiration = new Date(Date.now() + 60000) // 1 minute
+    ephemeralMessage = Authenticator.getEphemeralMessage(ephemeralAccount.address, expiration)
+    signature = await mainAccount.signMessage(ephemeralMessage)
+
+    authChain = [
+      {
+        type: AuthLinkType.SIGNER,
+        payload: mainAccount.address,
+        signature: ''
+      },
+      {
+        type: AuthLinkType.ECDSA_PERSONAL_EPHEMERAL,
+        payload: ephemeralMessage,
+        signature: signature
+      }
+    ]
+  })
+
+  describe('when an auth chain is not provided', () => {
+    it('should respond with an invalid response message indicating that the auth chain is required', async () => {
+      const response = await desktopClientSocket.emitWithAck(MessageType.REQUEST, {
+        method: 'method',
+        params: []
+      })
+
+      expect(response).toEqual({
+        error: 'Auth chain is required'
+      })
+    })
+  })
+
+  describe('when an auth chain is provided', () => {
+    it('should respond with a request response message', async () => {
+      const requestResponse = await desktopClientSocket.emitWithAck(MessageType.REQUEST, {
+        method: 'method',
+        params: [],
+        authChain
+      })
+
+      expect(requestResponse).toEqual({
+        requestId: expect.any(String),
+        expiration: expect.any(String),
+        code: expect.any(Number)
+      })
+    })
+
+    it('should return the sender derived from the auth chain on the recover response', async () => {
+      const requestResponse = await desktopClientSocket.emitWithAck(MessageType.REQUEST, {
+        method: 'method',
+        params: [],
+        authChain
+      })
+
+      const recoverResponse = await authDappSocket.emitWithAck(MessageType.RECOVER, {
+        requestId: requestResponse.requestId
+      })
+
+      expect(recoverResponse.sender).toEqual(mainAccount.address.toLowerCase())
+    })
+
+    describe('when the payload on the signer link does not match the address of the ephemeral message signer', () => {
+      let otherAccount: ethers.HDNodeWallet
+
+      beforeEach(() => {
+        otherAccount = ethers.Wallet.createRandom()
+
+        authChain[0].payload = otherAccount.address
+      })
+      it('should respond with an invalid response message, indicating that the expected signer address is different', async () => {
```

Comment:
Just a new line between the describe and the it:
```suggestion

      it('should respond with an invalid response message, indicating that the expected signer address is different', async () => {
```